code/blake2/Hacl.Impl.Blake2.Core.fsti

```diff
@@ -99,20 +99,16 @@ let blake2_params_v (#a: Spec.alg) (h: mem) (p: blake2_params a): GTot (Spec.bla
     (as_seq h p.salt)
     (as_seq h p.personal)
 
-noextract inline_for_extraction
-val create_default_params: a:Spec.alg ->
-  salt: lbuffer uint8 (salt_len a) ->
-  personal: lbuffer uint8 (personal_len a) ->
-  Stack (blake2_params a)
-    (requires fun h -> live h salt /\ live h personal /\
-      as_seq h salt == Seq.create (Spec.salt_length a) (u8 0) /\
-      as_seq h personal == Seq.create (Spec.personal_length a) (u8 0)
-    )
-    (ensures (fun h0 p h1 ->
-      h0 == h1 /\
-      blake2_params_loc p == loc salt `union` loc personal /\
-      blake2_params_inv h1 p /\
-      blake2_params_v h1 p == Spec.blake2_default_params a))
+inline_for_extraction noextract
+val alloca_default_params: a: Spec.alg -> StackInline (blake2_params a)
```

Comment:
I can't remember the precise issue I had in the previous versions that prevented me from doing this, but I'm glad to see the extra indirection gone.